ocaml/gencert/dune

```diff
@@ -41,6 +39,12 @@
   )
 )
 
+(install
+  (files gencert)
+  (section libexec_root)
+  (package stunnel)
```

Comment:
Can this change be explained in the commit?

Reply:
I've kept it in the xapi package in the end, where it originally was: I managed to adjust `libexec_root` such that this file ended up in the proper place.